addon/components/o-s-s/social-media-badge.stories.js

```diff
@@ -0,0 +1,87 @@
+import hbs from 'htmlbars-inline-precompile';
+
+const PostTypes = [
+  'article',
+  'facebook_status',
+  'instagram_media',
+  'tiktok_video',
+  'story',
+  'tweet',
+  'pin',
+  'youtube_video',
+  'twitch_stream'
+];
```

Comment:
Look at how we do things for typing in OSS::Button.

I suggest we use something like :
```typescript

export type SocialMediaType = 'article' | 'facebook_status' | 'instagram_media';
type skinDefinitionType = { [key in SocialMediaType]: string };
export const skinMatching: skinDefinitionType = {
  article: 'fab fa-wordpress',
  facebook_status: 'fab fa-facebook',
  instagram_media: ''
};
```